packages/dashboard-v2/README.md

```diff
@@ -8,3 +8,18 @@ This is a Gatsby application. To run it locally, all you need is:
 
 - `yarn install`
 - `yarn start`
+
+## Accessing remote APIs
+
+To be able to log in on a local environment with your production credentials, you'll need to make the browser believe you're actually on the same domain, otherwise browser will block the session cookie.
```

Comment:
```suggestion
To be able to log in on a local environment with your production credentials, you'll need to make the browser believe you're actually on the same domain, otherwise the browser will block the session cookie.
```